pyglet/gui/widgets.py

```diff
@@ -125,6 +136,17 @@ def __init__(self, x, y, pressed, depressed, hover=None, batch=None, group=None)
 
         self._pressed = False
 
+    # Override
+    @property
+    def value(self):
+        return self._pressed
+    
+    # Override
+    @value.setter
+    def value(self, value):
+        self._pressed = value
```

Comment:
It might be a good idea to add assertions in the setters, to prevent strange bugs. 
Something like: 
```py
assert type(value) is bool, "Value must be True or False."
or
assert type(value) in (int, float), "Value must be an int or float."
```